bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/Accumulation/AccumulatedSelection.cs

```diff
@@ -0,0 +1,194 @@
+using ElectionGuard.Decryption.Shares;
+using ElectionGuard.Decryption.Extensions;
+using ElectionGuard.Ballot;
+using ElectionGuard.Guardians;
+using ElectionGuard.ElectionSetup.Extensions;
+using ElectionGuard.ElectionSetup;
+
+namespace ElectionGuard.Decryption.Accumulation;
+
+/// <summary>
+/// accumulated collection of all guardian shares for a given selection.
+/// This object is used to compute the final selection value.
+///
+/// This is a mutable class that is filled in through the decrypion process.
+/// </summary>
+public record AccumulatedSelection : DisposableRecordBase, IElectionSelection
```

Comment:
a design choice i made was to put the proof on the accumulated selection. while it breaks mutability, this seemed like a logical place to put it in order to publish the election record since this object already contains most of what we need, and we also needed to capture the accumulated proof somewhere.